src/components/ContentHighlights/ContentHighlightArchivedAlert.jsx

```diff
@@ -0,0 +1,49 @@
+import { Alert } from '@edx/paragon';
+import { useContext, useState } from 'react';
+import { NEW_ARCHIVED_CONTENT_ALERT_DISMISSED_COOKIE_NAME } from './data/constants';
+import { EnterpriseAppContext } from '../EnterpriseApp/EnterpriseAppContextProvider';
+import { enterpriseCurationActions, isArchivedContent } from '../EnterpriseApp/data/enterpriseCurationReducer';
+
+const ContentHighlightArchivedAlert = () => {
+  const archivedContentCookies = {};
+  const [isOpen, setIsOpen] = useState(true);
+
+  const { enterpriseCuration: { enterpriseHighlightedSets, dispatch } } = useContext(EnterpriseAppContext);
+  enterpriseHighlightedSets.forEach(set => {
+    set.highlightedContent.forEach(content => {
+      if (isArchivedContent(content)) {
+        if (archivedContentCookies[set.uuid]) {
+          archivedContentCookies[set.uuid].push(content.contentKey);
+        } else {
+          archivedContentCookies[set.uuid] = [content.contentKey];
+        }
+      }
+    });
+  });
+
+  const setArchivedContentCookies = () => {
+    Object.entries(archivedContentCookies).forEach(([highlightSetUUID, courseKey]) => {
+      global.localStorage.setItem(
+        `${NEW_ARCHIVED_CONTENT_ALERT_DISMISSED_COOKIE_NAME}-${highlightSetUUID}`,
+        courseKey,
+      );
+    });
+    dispatch(enterpriseCurationActions.updateDismissedArchivedCourse(false));
+    setIsOpen(false);
+  };
+
+  return (
+    <Alert
+      variant="info"
+      dismissible
+      closeLabel="Dismiss"
```

Comment:
This is the default string for `Alert`; should be able to remove the prop if opting to use the default?